src/app/tests/TestInteractionModelEngine.cpp

```diff
@@ -223,6 +226,37 @@ void TestInteractionModelEngine::TestRemoveDuplicateConcreteAttribute(nlTestSuit
     InteractionModelEngine::GetInstance()->ReleaseAttributePathList(attributePathParamsList);
 }
 
+#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS && CHIP_CONFIG_SUBSCRIPTION_TIMEOUT_RESUMPTION
```

Comment:
are these enabled during testing?

Reply:
singe `chip_subscription_timeout_resumption = false` is the default, I would imagine this test will generally not be executed.